lib/routes/aibase/daily.ts

```diff
@@ -0,0 +1,129 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import { rootUrl, buildApiUrl } from './util';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/daily',
+    name: 'AI日报',
+    url: 'www.aibase.com',
+    maintainers: ['3tuuu'],
+    handler: async (ctx) => {
+        // 每页数量限制
+        const limit = Number.parseInt(ctx.req.query('limit') ?? '30', 10);
+        // 用项目中已有的获取页面方法，获取页面以及 Token
+        const currentUrl = new URL('discover', rootUrl).href;
+        const currentHtml = await ofetch(currentUrl);
+        const $ = load(currentHtml);
+        const logoSrc = $('img.logo').prop('src');
+        const image = logoSrc ? new URL(logoSrc, rootUrl).href : '';
+        const author = $('title').text().split(/_/).pop();
+        const { aILogListUrl } = await buildApiUrl($);
+        const response: NewsItem[] = await ofetch(aILogListUrl, {
+            headers: {
+                accept: 'application/json;charset=utf-8',
+            },
+            query: {
+                pagesize: limit,
+                page: 1,
+                type: 2,
+                isen: 0,
+            },
+        });
+        const items = await Promise.all(
+            response.data.slice(0, limit).map(async (item) => {
+                const articleUrl = `https://www.aibase.com/zh/news/${item.Id}`;
+                const description = await cache.tryGet(articleUrl, async () => {
```

Comment:
Cache the whole returned object of https://github.com/DIYgod/RSSHub/blob/09a376624c16839125763fee4d58c90b9233adbb/lib/routes/aibase/daily.ts#L49-L55.